lib/components/artist/artworks/grid.js

```diff
@@ -137,13 +139,15 @@ class ArtworksGrid extends React.Component {
           }
         }
 
-        const section = sectionedArtworks[sectionIndex]
-        section.push(artwork)
+        if (sectionIndex) {
```

Comment:
`if (sectionIndex) {` wasn't an awesome choice, as 0 is a legit section index ;)